app/views/application/_subnavigation.html.erb

```diff
@@ -16,10 +16,9 @@
         <% if current_organisation.ips.present? %>
           <%= link_to "Overview", root_path, class: active_tab(root_path) %>
         <% end %>
-        <%= link_to "Setting Up", setup_instructions_path, class: active_tab(setup_instructions_path) %>
-        <%= link_to 'Organisation', organisation_path(current_organisation), class: active_tab("organisation") %>
-        <%= link_to "Team members", team_members_path, class: "#{active_tab("team")}" + "#{active_tab("user")}"%>
+        <%= link_to "Setup Instructions", setup_instructions_path, class: active_tab(setup_instructions_path) %>
```

Comment:
Given we've had "the discussion", can we stick to single quotes when not interpolating?

Reply:
Took the decision to just fix this whole file, despite the noise it adds to the PR. 

I'd still be interested to hear opinions on this for future changes.